src/tests/ftest/launch.py

```diff
@@ -1406,12 +1406,18 @@ def get_yaml_data(yaml_file):
         dict: the contents of the yaml file
 
     """
+    class SafeLoaderIgnoreUnknown(yaml.SafeLoader):
+        def ignore_unknown(self, node):
```

Comment:
(pylint-missing-function-docstring) Missing function or method docstring
(pylint-unused-argument) Unused argument 'node'
(pylint-no-self-use) Method could be a function